packages/components/app/styles/components/dropdown.scss

```diff
@@ -278,61 +278,17 @@ $hds-dropdown-toggle-border-radius: $hds-button-border-radius;
 
 .hds-dropdown-list-item--variant-copy-item {
   width: 100%;
+  max-width: 100%;
```

Comment:
I don't think you need this (I checked in the showcase, and commenting it out doesn't do anything)

Reply:
I should have addressed this before I merged into the other branch, my apologies. I plan to address this in the feature branch. I think it might matter when it comes to long strings, but if it doesn't, I'll remove it.